packages/fether-react/src/Tokens/Tokens.js

```diff
@@ -39,67 +81,93 @@ class Tokens extends PureComponent {
       account: { address },
       history
     } = this.props;
+    const { canViewRecoveryPhrase, showWarning } = this.state;
 
     const backupAccountItem = {
       name: 'Backup Account',
       onClick: () => history.push(`/backup/${address}`)
     };
 
+    const backupPhraseItem = {
+      name: `${
+        showWarning ? 'Backup Recovery Phrase' : 'View Recovery Phrase'
+      }`,
+      onClick: () => history.push(`/backupPhrase/${address}/${showWarning}`),
+      warn: showWarning
+    };
+
     const menuItems = [
       {
         name: 'Add Tokens',
         onClick: () => history.push(`/whitelist/${address}`)
       }
     ];
 
-    if (this.isParitySignerAccount() === false) {
-      menuItems.unshift(backupAccountItem);
+    if (!this.isParitySignerAccount()) {
+      if (canViewRecoveryPhrase) {
+        menuItems.push(backupPhraseItem);
+      }
+      menuItems.push(backupAccountItem);
     }
 
     return menuItems;
   };
 
+  menuTrigger = () => {
+    const { showWarning } = this.state;
+
+    return (
+      <span style={{ display: 'flex' }}>
+        <Clickable className='icon -menu' />
+        {showWarning && <span className='icon -warning' />}
+      </span>
+    );
+  };
+
   render () {
     const {
       account: { address, name, type }
     } = this.props;
     const { isMenuOpen } = this.state;
 
+    const rightMenu = (
+      <div>
+        <MenuPopup
+          className='popup-menu-account'
+          horizontalOffset={1}
+          menuItems={this.menuItems()}
+          onClose={this.handleMenuClose}
+          onOpen={this.handleMenuOpen}
+          size='small'
+          trigger={this.menuTrigger()}
+        />
+      </div>
+    );
+
     return (
-      <React.Fragment>
```

Comment:
Please keep the `React.Fragement` (and keep the `nav class="footer-nav"` outside of the ` div class="tokens"` ) as this breaks the "sticky footer" behavior on this screen.